package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@supabase/supabase-js",
-  "version": "1.0.0",
+  "version": "0.0.0",
```

Comment:
@inian I think this is safe to revert back to 0.0.0? It's not used anywhere, right?

Reply:
Yeah semantic-release will update this automatically so this is unused.

Reply:
we might want to have semantic-release also check in this change